ts/src/bingx.ts

```diff
@@ -1700,14 +1691,14 @@ export default class bingx extends Exchange {
             } else if (isStopLossPriceOrder || isTakeProfitPriceOrder) {
                 // This can be used to set the stop loss and take profit, but the position needs to be opened first
                 if (isStopLossPriceOrder) {
-                    request['stopPrice'] = this.priceToPrecision (symbol, stopLossPrice);
+                    request['stopPrice'] = this.parseNumber (this.priceToPrecision (symbol, stopLossPrice));
```

Comment:
@Dan-krm we shouldn't use parseNumber upon building up the `request` because the return type of parseNumber is dynamic. we should be explicit and convert it to strings or floats here

Reply:
@carlosmiei would `parseFloat` be better here? Or is there a different approach where I can avoid the BadRequest errors and the Static test errors, I think parseFloat might still trigger some static test errors